### PR TITLE
Move order creation for embedded express sessions

### PR DESCRIPTION
**Pull request by andersem**
Use WooCommerce sessions table to initiate order created from callback

- Improve order creation with coupons
- Fix bug with different tax zone

### CODE REVIEW

File: includes/class-wc-dintero-hp-ajax.php [hunk 1]
```diff
@@ -300,272 +318,217 @@ public function update_shipping_line_id(){
...
 	}
+
 	/*
 	* The function can create order in woocommerce
-	* This funtion is used for backup callback if order does not get created in first instance
+	* This function is used for backup callback if order does not get created in first instance
 	*
 	*/
-	public function dhp_create_order(){
+	public function dhp_create_order() {
```
Comment (andersem) [inline includes/class-wc-dintero-hp-ajax.php:327]: Top-notch refactoring!

File: includes/class-wc-dintero-hp-checkout.php [hunk 2]
```diff
@@ -1127,14 +1128,14 @@ function validate(evt) {
 
 		}
 
-		if($sessionExpired || !$order_id){
+		// if($sessionExpired || !$order_id){
```
Comment (andersem) [inline includes/class-wc-dintero-hp-checkout.php:1131]: @chaikovskyia: I'm sure this is fine, but could you explain fast why this is commented out? If it should stay like it, please delete.

Comment (chaikovskyia) [reply includes/class-wc-dintero-hp-checkout.php:1131]: @andersem , this piece needs to be uncommented.

Comment (andersem) [reply includes/class-wc-dintero-hp-checkout.php]: :+1:

**Pull request merged**